src/view/channel-opening.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+import PropTypes from 'prop-types';
+import Background from '../component/background';
+import { Header, Title } from '../component/header';
+import { SmallButton, BackButton } from '../component/button';
+import MainContent from '../component/main-content';
+import { ResizeableSpinner } from '../component/spinner';
+import { H1Text, CopyText } from '../component/text';
+import Icon from '../component/icon';
+import { color } from '../component/style';
+
+//
+// Channel View
+//
+
+const styles = StyleSheet.create({
+  content: {
+    justifyContent: 'center',
+  },
+  spinnerWrapper: {
+    margin: 20,
+  },
+  bolt: {
+    height: 172 * 0.6,
+    width: 95 * 0.6,
+  },
+});
+
+const ChannelOpeningView = ({ channel, nav }) => (
+  <Background color={color.blackDark}>
+    <ChannelHeader
+      goChannelCreate={() => channel.initCreate()}
+      goHome={() => nav.goHome()}
+    />
+    <MainContent style={styles.content}>
+      <View style={styles.spinnerWrapper}>
+        <ResizeableSpinner
+          percentage={0.65}
+          size={190}
+          progressWidth={6}
+          gradient="openChannelsGrad"
+        >
+          <Icon image="lightning-bolt-gradient" style={styles.bolt} />
+        </ResizeableSpinner>
```

Comment:
The design has a grey circle behind it. Seems like we could add it behind the other one with position absolute.